packages/generator-joplin/generators/app/templates/webpack.config.js

```diff
@@ -67,10 +68,18 @@ function currentGitInfo() {
 	}
 }
 
+function validateCategories(categories) {
+		if ((categories.length !== new Set(categories).size)) throw new Error('Repeated categories are not allowed');
+		categories.forEach(category => {
+			if (!allPossibleCategories.includes(category)) throw new Error(`${category} is not a valid category. Please make sure that the category name is lowercase. Valid Categories are: \n${allPossibleCategories}\n`);
+		});
+}
+
 function readManifest(manifestPath) {
 	const content = fs.readFileSync(manifestPath, 'utf8');
 	const output = JSON.parse(content);
 	if (!output.id) throw new Error(`Manifest plugin ID is not set in ${manifestPath}`);
+	output.categories && validateCategories(output.categories);
```

Comment:
added this so that it only call it when categories are present

Reply:
that's a good point but I think you should do this check in validateCategories ("categories" being undefined is valid) otherwise you're doing validation in two places.

Reply:
Done :+1: